### PR TITLE
Clarify example https://issues.redhat.com/browse/ACM-3349

### PR DESCRIPTION
**Pull request by oafischer**
Clarify YAML example for ConfigMap. Also for 2.7.

### CODE REVIEW

**Review by dockerymick — APPROVED** [review]
Looks great! Thank you so much for taking this, approving

File: observability/manage_search.adoc [hunk 1]
```diff
@@ -56,6 +56,10 @@ data:
        - certificatepolicies
 ----
 
+The example allows `services` and `pods` to be collected from all `apiGroups`, while allowing all resources to be collected from the `admission.k8s.io` and `authentication.k8s.io` `apiGroups`. The example also prevents the central collection of `secrets` from all `apiGroups` while preventing the collection of `policies`, `iampolicies`, and `certificatepolicies` from the `apiGroup` `admission.k8s.io`.
```
Comment (Anxhela21) [inline observability/manage_search.adoc]:
```suggestion
The sample configmap above allows `Services` and `Pods` to be collected from all `apiGroups`, while allowing all resources to be collected from the `admission.k8s.io` and `authentication.k8s.io` `apiGroups`. Concurrently, it prevents the central collection of `Secrets` from all `apiGroups` while preventing the collection of `Policies`, `Iampolicies`, and `Certificatepolicies` from the `apiGroup` `admission.k8s.io`.
```

**Review by Anxhela21 — APPROVED** [review]
Left a minor comment - otherwise looks great! Thank you @oafischer !
/lgtm
/hold

File: observability/manage_search.adoc (hunk 1 above)
Comment (oafischer) [reply observability/manage_search.adoc]: Thanks, made some updates using your suggestions. To conform with IBM Style, the resources need to remain lower case as they appear in the example though.

**Review by Anxhela21 — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**